IPython/extensions/autoreload.py

```diff
@@ -116,15 +116,14 @@
 import types
 import weakref
 from importlib import import_module
-
+from IPython.utils.py3compat import PY3
```

Comment:
I cannot find it. If you are okay, please merge. Will cleanup once all are merged .